test/api/package/npm/delete-ownership.test.js

```diff
@@ -101,7 +118,34 @@ test('DELETE `/package/npm/ownership` 200 success', async (t) => {
 
   // the package they were co-maintaining is now fully the other person's
```

Comment:
comment should be updated as this is no longer covering the "only one person left" thing. speaking of which, is there still a test case for "only one person left"?